tests/subsys/fs/fs_api/src/test_fs_dir_file.c

```diff
@@ -332,9 +401,21 @@ void test_file_open(void)
 	zassert_not_equal(ret, 0, "Open a NULL file");
 
 	TC_PRINT("\nOpen a file with wrong abs path\n");
-	ret = fs_open(&filep, "/test_file.txt", FS_O_READ);
+	ret = fs_open(&filep, "/", FS_O_READ);
+	zassert_not_equal(ret, 0, "Open a file with wrong path");
+
+	TC_PRINT("\nOpen a file with wrong path\n");
+	ret = fs_open(&filep, "test_file.txt", FS_O_READ);
 	zassert_not_equal(ret, 0, "Open a file with wrong path");
 
+	TC_PRINT("\nOpen a file with wrong abs path\n");
+	ret = fs_open(&filep, "/test_file.txt", FS_O_READ);
+	zassert_not_equal(ret, 0, "Open a file with wrong abs path");
```

Comment:
Reorder these so that there would be no need for two "Open a file with wrong abs path" "headers".